sdk/metric/internal/instrument/descriptor.go

```diff
@@ -0,0 +1,24 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package instrument // import "go.opentelemetry.io/otel/sdk/metric/internal/instrument"
+
+import "go.opentelemetry.io/otel/sdk/instrumentation"
+
+type Description struct {
```

Comment:
Please add comments here. I take it this will grow, presently has just the fields needed for the stuff tested in this PR? I expect it will contain Unit eventually.

Reply:
This was moved to view.Instrument, and a comment was added.